go-ethereum-on-ubuntu/azuredeploy.parameters.json

```diff
@@ -0,0 +1,21 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newStorageAccountName": {
+      "value": "gethtest01"
+    },
+    "vmDnsName": {
+      "value": "publicNode"
```

Comment:
Can you change this value to "#####". That allows us to test this with an automated deployment bot as it can be replaced with an auto generated string